regulations/views/chrome.py

```diff
@@ -175,8 +175,10 @@ def check_tree(self, context):
 
         interp = generator.get_tree_paragraph(reg_part + '-Interp', version)
         if not interp:
-            raise error_handling.MissingSectionException(label_id, version,
-                                                         context)
+            interp = generator.get_tree_paragraph(label_id, version)
```

Comment:
Looking [at urls.py](https://github.com/cfpb/regulations-site/blob/master/regulations/urls.py#L82), this view only gets called if the URL pattern matches e.g. "/201-Subpart-A-Interp/2013-10706". Don't we already know that the label always ends in "-Interp"? Instead of `split`ting and then appending `-Interp`, could we just always get the paragraph for the label as is? Or is it the typical use case where we want to transform "201-Subpart-A-Interp" back to "201-Interp"?